lib/gateway/Shard.js

```diff
@@ -1220,7 +1047,6 @@ class Shard extends EventEmitter {
                 this.checkReady();
                 break;
             }
-            case "RESUMED":
```

Comment:
wait what we need this

Reply:
little mistake here, will be fixed in the next commit 👍